<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>590</width>
    <height>400</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="tabPosition" >
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRemotes" >
      <attribute name="title" >
       <string>Remotes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="title" >
           <string>Remotes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5" >
           <item>
            <widget class="QListView" name="remoteslistView" >
             <property name="toolTip" >
              <string>A remote file describes how to translate raw receiver codes to well known key names.</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" >
             <item>
              <spacer name="horizontalSpacer" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="btnRemoteDownload" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnRemoteNew" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnRemoteEdit" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnRemoteRemove" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox" >
          <property name="title" >
           <string>Controlled Desktop Profiles</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2" >
           <item>
            <widget class="QListView" name="desktopRemoteView" />
           </item>
           <item>
            <widget class="QCheckBox" name="chkControlAll" >
             <property name="toolTip" >
              <string>Control always all desktop profiles with the selected remote</string>
             </property>
             <property name="text" >
              <string>Controll All</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProfiles" >
      <attribute name="title" >
       <string>Profiles</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QSplitter" name="splitter_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QGroupBox" name="groupBox_11" >
          <property name="title" >
           <string>Desktop Profiles</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8" >
           <item>
            <widget class="QListView" name="desktoplistView" />
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" >
             <item>
              <spacer name="horizontalSpacer" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="btnDPDownload" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnDPNew" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnDPEdit" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnDPRemove" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_4" >
          <property name="title" >
           <string>Application Profiles</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6" >
           <item>
            <widget class="QListView" name="applicationlistView" >
             <property name="toolTip" >
              <string>An application profile describes the liri execution engine
how to contact the application and link application commands to well-known key names.</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" >
             <item>
              <spacer name="horizontalSpacer" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="btnAPDownload" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnAPNew" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnAPEdit" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnAPRemove" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
